e2e-tests/portal/cypress/e2e/journeys/checker/dashboard/dashboard.spec.js

```diff
@@ -60,13 +60,13 @@ context('View dashboard deals as a checker', () => {
       BANK1_DEALS.push(createdDeal);
     });
 
-    cy.insertOneGefApplication(GEF_DEALS.READY_FOR_CHECK, BANK1_MAKER1).then((gefDeal) => {
+    cy.insertOneGefApplication(GEF_DEALS.READY_FOR_CHECK, ADMIN).then((gefDeal) => {
```

Comment:
Seems that this should be `BANK2_MAKER2` instead of `ADMIN`? I don't see why `ADMIN` is required here, unrelated to the test description, _"View dashboard deals as a checker"_?

Reply:
With `BANK1_MAKER1` the test is unable to create deal so changed to `ADMIN` but when testing it logs as `BANK1_MAKER1`.
  
![Capture](https://github.com/user-attachments/assets/cf9b254b-cb1b-4711-9b3a-552ed43e398d)

Reply:
What does `signInLinkResponse` return? Bit odd that `insertGefApplication`  (that calls the `/v1/gef/application`) endpoint is not working with `BANK1_MAKER1`.

I can't see how this could happen with these changes, as in the API only the `/user`routes have changed, unless i'm missing something

Reply:
`signInLinkResponse` returns user details as `maker1@ukexportfinance.gov.uk` with `maker` role. With `maker` role user is trying to access end point `http://localhost:5001/v1/users` but only `admin` have access to this so the response received is `401: Unauthorised You do not have access to this page`. Hence the test fails.
![Capture](https://github.com/user-attachments/assets/a2331a79-36a7-4df9-a0df-3ac33f10fd9e)